src/server/payment/simulation.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const request = require('request-promise-native');
+const authSupport = require('../auth/authsupport');
+
+let config;
+
+const setup = navConfig => {
+    config = navConfig;
+    return {
+        simulate: simulate,
+        isValid: isValid
+    };
+};
+
+const simulate = async (vedtak, accessToken) => {
+    if (!isValid(vedtak)) {
+        return Promise.reject(`Invalid vedtak supplied. Should look like '${validVedtak}'`);
+    }
+
+    vedtak.saksbehandler = authSupport.nameFrom(accessToken);
+
+    const options = {
+        uri: config.spennUrl,
+        headers: {
+            Authorization: `Bearer ${accessToken}`
+        },
+        body: vedtak,
+        json: true
+    };
+
+    return request.post(options);
+};
+
+const isValid = vedtak => {
+    const requiredKeys = Object.keys(validVedtak);
+    const actualKeys = Object.keys(vedtak);
+    return (
+        actualKeys.length == requiredKeys.length &&
```

Comment:
Klikker $penn hvis man sender med felter den ikke bryr seg om?

Reply:
Det vet jeg ikke, men det er ikke god folkeskikk å sende bonusmateriale (pluss at man åpner for potensielle sikkerhetsproblemer med å la andre injecte ting inn i objektene sine)

Reply:
Spenn skal i prinsippet ikke klikke, men jeg istemmer jan kåre. ikke noe poeng å sende oss ting vi ikke trenger :)

Reply:
Enig i at det ikke er noe poeng å sende ekstramateriale. Jeg har nok en liten hangup på å være fleksibel i hva man tar mot - her kunne vi ha fisket ut de verdiene vi har lyst til å sende til Spenn. Men no biggie.

Reply:
Litt apropos, og det er vel ikke nødvendigvis helt analogt (request kontra response), men det er noe rundt robusthet mot endringer i omkringliggende systemer her.. 
![image](https://user-images.githubusercontent.com/136872/65944803-cec49c00-e432-11e9-97ea-49ab860f8257.png)
https://nav-it.slack.com/archives/C8MU0M6L8/p1569912920053800